File: Rregulloje.UI/RregullojeWebAPI/src/app/_ViewModels/UserMessageEmailViewModel.ts
export class UserMessageEmailViewModel {
  id: number;
  name: string;
  fromEmail: string;
  toEmail: string;
  subject: string;
  message: string;

  constructor(id: number = null, name: string = null, fromEmail: string = null, toEmail: string = null, subject: string = null, message: string = null) {
    this.id = id;
    this.name = name;
    this.fromEmail = fromEmail;
    this.toEmail = toEmail;
    this.subject = subject;
    this.message = message;
  }
}

File: Rregulloje.Application/ViewModels/UserMessageEmailViewModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Rregulloje.Application.ViewModels
{
    public class UserMessageEmailViewModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string FromEmail { get; set; }
        public string ToEmail { get; set; } 
        public string Subject { get; set; }
        public string Message { get; set; } 
    }
}
